crates/host/src/wasmbus/mod.rs

```diff
@@ -495,12 +509,42 @@ impl Lattice {
             })
             .collect()
             .await;
+        let providers: Vec<_> = self
+            .providers
+            .read()
+            .await
+            .iter()
+            .flat_map(
+                |(
+                    public_key,
+                    Provider {
+                        claims, instances, ..
+                    },
+                )| {
+                    instances.keys().map(move |link_name| {
+                        if let Some(jwt::CapabilityProvider { capid, .. }) = &claims.metadata {
```

Comment:
Do you need to duplicate the json macro, or can you move the capid to its own variable to make the branch logic smaller?